misc/python/materialize/mzcompose/services.py

```diff
@@ -99,6 +99,9 @@ def __init__(
                 "persist_sink_minimum_batch_updates=128",
                 "enable_multi_worker_storage_persist_sink=true",
                 "storage_persist_sink_minimum_batch_updates=100",
+                "persist_pubsub_push_diff_enabled=true",
+                "persist_pubsub_client_enabled=true",
+                "enable_monotonic_oneshot_selects=true",
```

Comment:
This is currently a manual selection out of the existing flags. Further boolean flags exist, but we don't want to enable them yet?

Reply:
Regarding the fact that it is a manual process , in https://github.com/MaterializeInc/materialize/pull/19120 I am arguing that all flags should be enabled by default in the CI , without having to be explicitly listed. Let's see how far I go.

Reply:
enable_ld_rbac_checks and enable_rbac_checks will probably be quite annoying to enable because of lacking permissions. Let me give it a try.

Reply:
[windows_functions](https://app.launchdarkly.com/default/production/features/window_functions/targeting) and [enable_auto_route_introspection_queries](https://app.launchdarkly.com/default/production/features/enable_auto_route_introspection_queries/targeting) would be further ones.

Shall we also explicitly add `enable_ld_rbac_checks` with value `false` to have a complete list?

Reply:
Well in #19120 the consensus was that the CI should not run with all flags enabled. So my plan was to push this PR as is in order to get coverage on the important bits and then we can discuss again during the next QA meeting. Something along the lines of:

- we still want the product-wide, or at least source-wide flags enabled in the CI in a way that will cause Zippy and Platform Checks to always run with those flags enabled
- the SQL-side specific flags, such as rbac and window_functions can then be left to dedicated tests to exercise.